Automattic/wp-calypso
# Fix quick action styling when jetpack is not connected

<img width="476" alt="Screenshot 2567-06-18 at 17 53 47" src="https://github.com/Automattic/wp-calypso/assets/6851384/a39334a2-daa8-4afc-ab7f-95aac62c1913">

I stumbled upen this when testing a site with a broken Jetpack connection, which is currently the only way to reproduce it. As there is no connection, `activeThemeData` is unavailable, so there is no link, resulting in a `button` output instead of a `a`. The button does nothing so I have disabled it and fixed the styling. 

## Proposed Changes

https://github.com/Automattic/wp-calypso/assets/6851384/8d69197b-c585-41f1-b89e-f84091a645b3


* Fix styling and disable the button when no editor link is available. I used `opacity: .5` as that's common for `disabled` elements. 


## Why are these changes being made?



## Testing Instructions



* I have a broken Jetpack site, but if you don't, you can null the `href` instead
* Reproduce then apply the PR and see it's fixed
* Verify other links continue to work

## Pre-merge Checklist
